### PR TITLE
Fix/noise zones

### LINKED ISSUE #449: Modeller exception: LinkPrinting run ended with exception
When using Print links from HELMET toolbox on ve0:

Modeller Exception
LinkPrinting run ended with exception: local variable 'zone_width' referenced before assignment

### PR DESCRIPTION
**Pull request by johpiip**
Fixes #449

### CODE REVIEW

File: Scripts/utils/calc_noise.py [hunk 1]
```diff
@@ -75,4 +75,6 @@ def calc_noise(self, link):
             if interval[0] <= start_noise < interval[1]:
                 zone_width = func[interval](start_noise - interval[0])
                 break
+        else:
+            raise ValueError("{}".format(link.id))
```
Comment (zptro) [inline Scripts/utils/calc_noise.py:79]:
```suggestion
            raise ValueError("No noise interval found for link {}".format(link.id))
```

**Review by zptro — CHANGES_REQUESTED** [review]
The problem could be solved by setting `traffic` to at least 0.01 here:

https://github.com/HSLdevcom/helmet-model-system/blob/f43f167ca5caae9aa2ee776e9b35c9b46d19a37f/Scripts/utils/calc_noise.py#L36

instead of here:

https://github.com/HSLdevcom/helmet-model-system/blob/f43f167ca5caae9aa2ee776e9b35c9b46d19a37f/Scripts/utils/calc_noise.py#L46

Then ´cross_traffic` would also be at least 0.01, which would prevent noise from going below zero.

**Review by zptro — APPROVED** [review]

**Comment by johpiip** [comment]
Nice! I forgot about this when I had to hurry assessments but great that you did not. :) Should we make a test run or did you do that already?

**Comment by zptro** [comment]
> Nice! I forgot about this when I had to hurry assessments but great that you did not. :) Should we make a test run or did you do that already?

Yes, we should probably do a 2040 test run.

**Comment by johpiip** [comment]
@zptro I tested this and it works! Those links which had a noise zone width of 5 still have it. Ve0 scenario does not crash when writing links.

**Pull request merged**